src/main/java/core/basesyntax/Drawable.java

```diff
@@ -0,0 +1,5 @@
+package core.basesyntax;
+
+public interface Drawable {
+    double area();
+}
```

Comment:
The interface name 'Drawable' does not accurately represent the behavior it provides. According to the checklist, interface names should not be verbs. Moreover, this interface should not contain the 'area()' method as it is conceptually different from drawing. Consider splitting the responsibilities into two separate interfaces.